awxkit/requirements.txt

```diff
@@ -1,8 +1,4 @@
 PyYAML>=5.1
-cryptography
```

Comment:
might want to confirm awxkit tests don't need this

Reply:
doesn't look to me like they do